headers/cub3d.h

```diff
@@ -174,6 +190,12 @@ void	my_destroy_image(void *mlx, void *img);
 char	*skip_whitespaces(char *str);
 int		convert_rgb(char *rgb);
 
+// utils_rays.c
+float	horizontal_up(t_info *info, float hit[2], float dx, float dy);
+float	horizontal_down(t_info *info, float hit[2], float dx, float dy);
+float	vertical_right(t_info *info, float hit[2], float dx, float dy);
+float	vertical_left(t_info *info, float hit[2], float dx, float dy);
```

Comment:
To be more consistant with the rest of the code, we could use array for delta
```suggestion
float	horizontal_up(t_info *info, float hit[2], float delta[2]);
float	horizontal_down(t_info *info, float hit[2], float delta[2]);
float	vertical_right(t_info *info, float hit[2], float delta[2]);
float	vertical_left(t_info *info, float hit[2], float delta[2]);
```

Reply:
Very good idea !

Reply:
Done: eee14faa5865180a8d97b2d8fce6f10eaf627448